packages/dashboard/src/components/DetailPanel/Header/Header.tsx

```diff
@@ -49,12 +49,15 @@ const header: React.SFC<DetailPanelHeaderProps> = (props: DetailPanelHeaderProps
   };
 
   const _renderElement = () => {
-    const { title, personaHeader } = props;
+    const { title, personaHeader, imageAlt, imageInitials, imageUrl } = props;
 
     if (personaHeader) {
       return (
         <div className={css.personaMode}>
           <Persona
+            imageAlt={imageAlt}
+            imageUrl={imageUrl}
+            imageInitials={imageInitials}
```

Comment:
do you want to support aria hidden so narrator does not read this image if need be

Reply:
so, basically, header has another mode which accepts the jsx element, dev can just compose a own header with all these features and apply to it